pages/common/mongorestore.md

```diff
@@ -0,0 +1,19 @@
+# mongorestore
+
+> Utility to import a collection or database from a binary dump into a MongoDB instance.
+
+- Import a bson data dump from a folder to a mongodb instance:
+
+`mongorestore --db {{database_name}} {{path/to/folder}}`
+
+- Import a bson data dump from a folder to a given mongodb database host running at a given port with user authentication; user will be prompted for password:
```

Comment:
This is a bit of a mouthful. We should harness punctuation to better guide reading flow, e.g.

`- Import a bson data dump from a folder to a given mongodb database host, running at a given port, with user authentication (user will be prompted for password):`